templates/disk-uploader/examples/disk-uploader-taskrun.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: tekton.dev/v1
+kind: TaskRun
+metadata:
+  name: disk-uploader-taskrun
+spec:
+  taskRef:
+    name: disk-uploader-task
+  params:
+  - name: EXPORT_SOURCE_KIND
+    value: vm
+  - name: EXPORT_SOURCE_NAME
+    value: example-vm
+  - name: VOLUME_NAME
+    value: example-dv
+  - name: IMAGE_DESTINATION
+    value: quay.io/boukhano/example-vm-exported:latest
```

Comment:
use kubevirt-tekton-tasks repo instead your private

Reply:
I see that `quay.io/kubevirt-tekton-tasks` doesn't exist, do you mean to use `quay.io/kubevirt` instead?

Reply:
correct :+1: